doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -701,6 +708,22 @@ Libraries for Zigbee
       * Crypto library used for performing software AES ecryption.
         Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.
 
+  * ref:`lib_zigbee_fota`:
+
+    * Added:
+
+      * Support for the combined application and network core updates for the nRF5340.
```

Comment:
```suggestion
      * Support for the combined application and network core updates for the nRF5340 DK.
```

Reply:
Why DK?
The library allows you to upgrade any board with nRF5340.

Reply:
Fixed